### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by gerroon** [leomoon-studios/blender-light-studio#58]
Hi

I am trying to use this addon with Blender 3.0

```
location: <unknown location>:-1
Error: Python: Traceback (most recent call last):
  File "C:\Users\USER\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\leomoon-lightstudio\light_operators.py", line 189, in execute
    bpy.ops.lls_list.new_profile()
  File "C:\BLENDER\3.0\scripts\modules\bpy\ops.py", line 132, in __call__
    ret = _op_call(self.idname_py(), None, kw)
RuntimeError: Error: Python: Traceback (most recent call last):
  File "C:\Users\USER\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\leomoon-lightstudio\light_profiles.py", line 109, in execute
    lls_collection = get_lls_collection(context)
  File "C:\Users\USER\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\leomoon-lightstudio\common.py", line 25, in get_lls_collection
    return [c for c in context.scene.collection.children if c.name.startswith('LLS')][0]
IndexError: list index out of range
location: C:\BLENDER\3.0\scripts\modules\bpy\ops.py:132
location: <unknown location>:-1

```

**Comment by leomoon** [comment]
Hello,

Blender 3.0 is still in alpha stages and it changes a lot. We've already issues a fix a few weeks ago and it is broken again.

We will make a release when Blender 3.0 final is out.

**Comment by leomoon** [comment]
this is fixed in LightStudio 2.10.0
